### DISCUSSION TITLE
2010/09/20/%E5%9B%9E%E8%B0%83vi

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#296] [Announcements]
# 2010/09/20/%E5%9B%9E%E8%B0%83vi

LabVIEW界面程序最常用的结构就是循环事件结构。用事件结构截获用户在界面上对控件的操作，然后做出相应处理。

https://blog.qizhen.xyz/2010/09/20/%E5%9B%9E%E8%B0%83vi

**Comment by ruanqizhen** [comment]
之前博客下的留言：

駐
2011年01月30日 16:28 编辑
又学了一招，谢谢分享

回复

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Jucao
2010年09月26日 12:01 编辑
恭喜了，Qizhen，要升级了！
